pulsar-client-tools/src/main/java/org/apache/pulsar/shell/PulsarShell.java

```diff
@@ -216,7 +216,6 @@ public void reload(Properties properties) throws Exception {
     }
 
     public void run() throws Exception {
-        System.setProperty("org.jline.terminal.dumb", "true");
```

Comment:
what's the impact on Linux and Mac ?

Reply:
Actually nothing. I had firstly misunderstood the meaning of this property. It's not useful